app/gui2/src/util/rect.ts

```diff
@@ -9,6 +9,10 @@ export class Rect {
     readonly size: Vec2,
   ) {}
 
+  static fromBounds(left: number, top: number, right: number, bottom: number): Rect {
```

Comment:
Didn't know the convention. `Array` does not seem to follow it.

Reply:
ah, yeah, i meant the convention in this codebase. but yeah, given that it's not the JS convention, it might be a better idea to rename everything to `camelCase` in that case.

Reply:
Yeah, I started the convention of using `PascalCase` for custom "constructors", things that would return `Self` in Rust. Didn't really thought about it at any point, it just seemed natural to me for some reason. In this PR I think we should stick to it though. If you think this is incorrect/inconsistent with the rest of the codebase, I'm OK with changing that convention, but then it should be done separately with other formatting changes.